llvm/test/Transforms/DropUnnecessaryAssumes/basic.ll

```diff
@@ -1,6 +1,9 @@
 ; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 6
 ; RUN: opt -S -passes=drop-unnecessary-assumes < %s | FileCheck %s
 
+declare void @use(i32 %x)
```

Comment:
Could use llvm.fake.use?

Reply:
This is an intrinsic related to debug info preservation, I don't think we should use it more generally.